device_id: 0
camera_mode: 0
flip_method: 0
frame_rate: 20
image_width: 1280
image_height: 720
image_topic: "/camera_0"
